### DISCUSSION TITLE
[Feature request]: simulation output

### DISCUSSION THREAD

**Discussion by Somy2024** [OpenTOPAS/OpenTOPAS#43] [Run-Problem]
### Feature request

Hi ,I just installed openTOPAS but I am facing problem in output, as it is just empty dark background. What would be the reason, and how can i fix it? 

### TOPAS category

Outcome model

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
Please provide your parameter file or a screenshot of the simulation log output.

**Reply by Somy2024** [reply]
![Screenshot 2024-07-05 120615](https://github.com/OpenTOPAS/OpenTOPAS/assets/155381603/415775d7-69cd-4181-8218-8e1c37a66115)
this is the screenshot of running  cd ~/Applications/TOPAS/OpenTOPAS-install/examples/Graphics
   topas QtTest.txt

**Reply by rezaamya** [reply]
Same here. I followed all steps using this link:
https://opentopas.readthedocs.io/en/latest/getting-started/WSL.html

At the end, I am facing with a black screen. I even tested other files like "topas TestColor.txt" and the output is same.

**Reply by icmeyer** [reply]
Hi,
It looks like this is a QT error. Can you confirm that you installed the QT packages by running:
`    sudo apt-get install -y qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools`

The two errors 
```
MESA: error: ZINK: failed to choose pdev
glx: failed to create drisw screen
```
Show some WSL specific results when doing a google search. If you are comfortable, you may want to try some of the recommended fixes out there, and do let us know if some tweak fixes the problem!

Maybe someone with more WSL experience can chime in

**Comment by silongGG** [comment]
Hi there,
I experienced the same problem and fixed it by upgrading Mesa. I am running Ubuntu 2404 in VirtualBox.
The following code fixed my problem:

sudo add-apt-repository ppa:oibaf/graphics-drivers
sudo add-apt-repository ppa:kisak/kis
sudo apt upgrade
sudo apt update

Hope this helps you!

Jianan
